src/ray/object_manager/format/om.fbs

```diff
@@ -0,0 +1,30 @@
+// Object Manager protocol specification
+
+enum OMMessageType:int {
+  PullRequest = 1
+}
+
+table PushRequest {
+
+}
+
+table PullRequest {
```

Comment:
I stared at this for a while.  I understand there might not be a better name, but we should try ;)